mitodl/rapid-response-xblock
# docs: Update readme with reports plugin info

#### What are the relevant tickets?
#121 

#### What's this PR do?
- Removes obsolete instructions in the readme
- Introduces a new installation option to install locally
- Adds the information about the rapid response reports plugin

#### How should this be manually tested?
Just check the Readme for clarity and grammar if any, or visit https://github.com/mitodl/rapid-response-xblock/blob/arslan/121-update-readme/README.md to see how things look after markdown.
